src/http/routing.rs

```diff
@@ -1530,6 +1544,18 @@ impl<'a> RouteInfo<'a> {
                 Route::GuildsIdRolesId(guild_id),
                 Cow::from(Route::guild_roles(guild_id)),
             ),
+            RouteInfo::EditVoiceState {
+                guild_id,
+                user_id,
+            } => {
+                let route = if let Some(user_id) = user_id {
+                    Cow::from(Route::guild_voice_states(guild_id, user_id))
+                } else {
+                    Cow::from(Route::guild_voice_states(guild_id, "@me"))
+                };
+
+                (LightMethod::Patch, Route::GuildsIdVoiceStates(guild_id), route)
+            },
```

Comment:
Not sure if this should be separated into two routes.  
`PATCH /guilds/:guild_id/voice-states/@me`
`PATCH /guilds/:guild_id/voice-states/:user_id` are separate endpoints so it should probably be separated to have their own rate limit buckets?

Reply:
> so it should probably be separated to have their own rate limit buckets?

Since they are separate, yes.